content/10.community_building.md

```diff
@@ -7,15 +7,15 @@ Choosing a software package for a particular purpose involves evaluating several
 
 There are several rough categories of individuals engaged in development and utilization of `yt`.
 As a result of its API-first design, there are few if any individuals who use `yt` that do not do so through the scripting interface; this means that the vast (if not exclusive) majority of individuals who interact with the functionality in `yt` are doing so by writing their own scripts, modules, and code, and arguably engaging in a value-added development process of their own.
-The majority of individuals using `yt` at present are in astronomy and astrophysics, typically fields of simulation, although there is an increasing group of individuals from other domains that are participating in development and using `yt` for their own domain-specific problems.
+The majority of individuals using `yt` at present are in astronomy and astrophysics, typically fields of simulation, although an increasingly large group of individuals from other domains is participating in development and using `yt` for their own domain-specific problems.
```

Comment:
While your change is grammatically correct: "group...is participating", it sounds awkward because of "individuals from other domains" are both plural which would tricks the ear into thinking it should be using "are".  I think this could be improved by replacing the text as:

"...typically fields of simulation, although representatives from other domains are increasingly participating in development and using `yt` for their own domain-specific problems."

...or something.

Reply:
works for me. AFAIK, "A bunch of blank is/are" are both corrects, but I may be applying French grammar involuntarily. I agree your suggestion works better anyway. Thanks !